docs/modules/ROOT/pages/ios_settings.adoc

```diff
@@ -0,0 +1,97 @@
+:keywords: settings, passcode lock, biometric lock, theme, logging, ownCloud, iOS, iPhone, iPad
+:description: This guide steps you through how to configure ownCloud's iOS App for iPhone and iPad. It covers security, theme, logging, and media upload settings.
+:heic-image-url: https://en.wikipedia.org/wiki/High_Efficiency_Image_File_Format
+
+To manage the settings in ownCloud's iOS App for iPhone and iPad, click btn:[Settings] in the bottom right-hand corner of the Accounts List view.
+
+image:user-accounts-list-annotated-with-callout.png[Accessing settings in ownCloud's iOS App for iPhone and iPad.]
+
+== Security
+
+=== Passcode 
+
+To protect access to the iOS app with a Passcode, enable btn:[Passcode Lock] in the Setting's Security section.
+You will then be prompted to enter, and repeat, a 4-digit Passcode.
+
+image:settings-security-passcode-enabled.png[Enable a Passcode lock in ownCloud's iOS App for iPhone and iPad.]
+
+=== Lock Delay
+
+When a Passcode is enabled, the app will be locked every time you change to another application.
+However, under menu:Settings[Security > Lock application], you can choose to only lock the application after 1, 5, or 30 minutes, instead of "_always_", which is the default.
+
+image:settings-lock-application-duration.png[Set the application lock duration in ownCloud's iOS App for iPhone and iPad.]
+
+=== Biometrical Lock
+
+After a Passcode has been created, a Biometrical Lock, or Touch ID, can also be used to gain access to the app.
+To enable it enable btn:[Touch ID] in the Setting's _Security_ section, and then enter your 4-digit Passcode.
+The next time you need to authorise access to the app, you will be able to enter either your Passcode, or use your stored biometrical data.
+
+image:authorise-access-with-passcode-or-biometric-data.png[Authorise access with passcode or biometric data in ownCloud's iOS App for iPhone and iPad.]
+
+=== Trusted Certificates
+
+==== View Previously Approved Certificates
+
+To view previously approved certificates, navigate to menu:Settings[Certificates] (for any one of your registered accounts), and you will see them listed in the "_User-Approved Certificates_" section.
+
+==== Inspect Previously Approved Certificates Details
+
+To view previously approved certificates, swipe left on any of the accounts in the accounts list and click btn:[Edit].
+Then, under "_SERVER URL_", click btn:[Certificate Details].
+You will then be able to see all of the certificate's details.
+
+==== Revoke Previously Approved Certificates
+
+image:83_Settings_certs.png[83_Settings_certs.png]
+
+To revoke one or more previously approved certificates, first navigate to menu:Settings[Certificates] (for any one of your registered accounts).
+Then, in the "_User-Approved Certificates_" section, swipe left on the certificate(s) that you wish to revoke and press btn:[Revoke approval].
+
+== Themes
+
+The iOS app comes with three themes: 
+
+* ownCloud Light
+* ownCloud Dark; and
+* ownCloud Classic
```

Comment:
Themes' names got rid of  `ownCloud` word, due to branding regards. Now they are `Light`, `Dark` and `Classic`.

Reply:
Solved with 6f7664a70be9091d2309da76c100d10d79bffc6f